<p>
Using the same value on either side of an operator is almost always a mistake.
It can be either a copy/paste error and therefore a bug, or it is simply wasted code, and should be simplified.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
if a = a then ...
</pre>