<p>
Specify the columns in an INSERT statement to guarantee that the code won't stop working when a new column is added.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
insert into tab values (1);
</pre>