<p>
The NVL function lets you replace a null value with a valid value value. So, don't specify a NULL literal or an empty string as a parameter of a NVL call.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
var := nvl(x, '');
</pre>