<p>
Duplicate a condition in a comparison is almost always a mistake.
It can be either a copy/paste error and therefore a bug, or it is simply wasted code, and should be simplified.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
IF (x = 1) AND (x = 1) THEN ...
</pre>